src/features/virtualized-table-renderers/readableTimeCellRenderer.js

```diff
@@ -6,7 +6,7 @@ import type { CellRendererParams } from './flowTypes';
 
 const readableTimeCellRenderer = (cellRendererParams: CellRendererParams) =>
     baseCellRenderer(cellRendererParams, cellValue => (
-        <ReadableTime timestamp={Date.parse(cellValue)} alwaysShowTime />
+        <ReadableTime timestamp={Number.isInteger(cellValue) ? cellValue : Date.parse(cellValue)} alwaysShowTime />
```

Comment:
Is this due to some casting done by the Date object? Wouldn't instanceof Date be more accurate?